src/body_storage.rs

```diff
@@ -56,44 +56,32 @@ impl<N: PtReal> BodyStorage<N> {
 }
 
 impl<N: PtReal> BodySet<N> for BodyStorage<N> {
-    type Body = dyn NpBody<N>;
```

Comment:
Why remove this type? Don't you think it's much better using it since you have less code to write.

Reply:
Thank you for reviewing this!

nphysics removed it, I think. The documentation for `BodySet` in `nphysics` 0.14.0 only lists `type Handle` in https://docs.rs/nphysics3d/0.14.0/nphysics3d/object/trait.BodySet.html.

I don't think we can keep it unless it is there in nphysics's definition too.